stable/gocd/templates/gocd-agent-deployment.yaml

```diff
@@ -127,6 +131,9 @@ spec:
           {{- end }}
           securityContext:
             privileged: {{ .Values.agent.privileged }}
+    {{- if .Values.agent.restartPolicy }}
```

Comment:
No need for the if condition, just have the restartPolicy in values.yaml to be `Always`. Which is the same as not specifying it. That way, you can avoid this check.

Reply:
good point. Changed.